roles/osg/templates/osg.env.j2

```diff
@@ -0,0 +1,5 @@
+TOKEN={{ osg_token }}
+GLIDEIN_Site=Chameleon
+GLIDEIN_ResourceName={{ osg_site_resource_name }}
+GLIDEIN_Start_Extra=True
+OSG_SQUID_LOCATION=http://127.0.0.1:3128
```

Comment:
Ah, this is the reason for "--net=host", I take it? OK. Does the OSG Squid thing run in a container as well? If so, we can connect them together and then this problem goes away.